harpoon

```diff
@@ -90,11 +118,13 @@ list_sessions() { tmux ls -F '#{session_name}=#{session_path}'; }
 bold() { printf "\033[1m%s\033[0m\n" "$*"; }
 
 main() {
-    while getopts ":haAr:ls:e" opt; do
+    while getopts ":haAo:O:r:ls:e" opt; do
         case "$opt" in
         h | :) help && exit 0 ;;
-        a) add ;;
-        A) add 1 ;;
+        a) add false ;;
+        A) add true ;;
+        o) overwrite false "$OPTARG" ;;
+        O) overwrite true "$OPTARG" ;;
```

Comment:
I would suggest we use `-r` and `-R` as in replace with session and replace with session:window:pane. We can use `-d` as in delete to free the current `-r` flag.